eventcontent.go

```diff
@@ -175,7 +175,7 @@ func (c *powerLevelContent) userLevel(userID string) int64 {
 }
 
 // eventLevel returns the power level needed to send an event in the room.
-func (c *powerLevelContent) eventLevel(eventType string, eventStateKey *string) int64 {
+func (c *powerLevelContent) eventLevel(eventType string, isState bool) int64 {
```

Comment:
It's not obvious at call-sites what `eventLevel(eventType, false)` means. Can we enum this please to make it obvious to callers please?